core/src/main/scala/org/http4s/blaze/channel/nio2/ClientChannelFactory.scala

```diff
@@ -48,7 +48,9 @@ final class ClientChannelFactory(
             s"An attempt to establish connection with $remoteAddress timed out after $connectTimeout.")
           val finishedWithTimeout = p.tryFailure(exception)
           if (finishedWithTimeout) {
-            try { ch.close() } catch { case NonFatal(_) => /* we don't care */ }
+            try {
+              ch.close()
+            } catch { case NonFatal(_) => /* we don't care */ }
```

Comment:
Do we know why it's more aggressively wrapping things now?  It's fine, but I shudder to think how large the diff is going to be on http4s core.

Reply:
I'll see if I can get the config to leave those alone.

Reply:
https://github.com/scalameta/scalafmt/issues/1437

Reply:
Yeah, that link was meant for the other comment.